app/controllers/validation_requests_controller.rb

```diff
@@ -10,16 +10,6 @@ def index
 
   def new; end
```

Comment:
Do we want this, also shall we remove these actions from [routes](https://github.com/unboxed/bops/blob/main/config/routes.rb#L62)?

Reply:
I did remove - look at the last commit https://github.com/unboxed/bops/pull/583/commits/4ad264dfde7cef634b780c65644dc5f20b441081

Reply:
I was going commit by commit so I missed it! Good job 😄